tests/unit/StaffNotation.spec.ts

```diff
@@ -502,6 +504,8 @@ describe('StaffNotation', () => {
     // verify that build and create methods were called
     expect(buildRestorationFiling).toHaveBeenCalled()
     expect(createFiling).toHaveBeenCalled()
+    await Vue.nextTick()
+    await Vue.nextTick()
```

Comment:
I was getting some weird behavior here. The test was returning the value of the previous one. The next test was returning the value of this test. This fixes that by waiting for the DOM to update.

Keep in mind I've tested this in local and it works fine!